crates/fee/src/tests.rs

```diff
@@ -13,3 +27,73 @@ fn should_get_issue_fee() {
         );
     })
 }
+
+#[test]
+fn should_set_issue_fee() {
+    run_test(|| {
```

Comment:
You could have put the `run_test` inside `test_setter` to save a couple of lines of code but that's alright :)